src/test/java/org/broadinstitute/hellbender/tools/exome/allelefraction/AlleleFractionSimulatedData.java

```diff
@@ -22,6 +23,11 @@
  * @author David Benjamin
  */
 public final class AlleleFractionSimulatedData {
+    public static final ReadCountCollection TRIVIAL_TARGETS = new ReadCountCollection(
```

Comment:
this seems a bit unsatisfying to me...is there no way to have an empty ReadCountCollection?